ui/src/literature/__tests__/citeArticle.test.js

```diff
@@ -21,14 +21,34 @@ describe('citeArticle', () => {
     done();
   });
 
-  it('sends request with Accept header based on format and returns text', async done => {
+  it('returns a status code that is not 2xx without data', async done => {
     const citeUrl = '/literature/12345';
-    const format = 'x-another';
+    const format = 'x-test';
     mockHttp
       .onGet(citeUrl, null, { Accept: 'application/x-test' })
-      .replyOnce(200, 'Test');
-    const content = await citeArticle(format, 12345);
-    expect(content).toEqual('Not supported yet :(');
+      .replyOnce(500);
+    let error;
+    try { 
+      await citeArticle(format, 12345);
+    } catch (e) {
```

Comment:
why not  `toThrow` instead of catching the error explicitly in the tests.

Reply:
I was having some problems with this. The error was uncatched so the execution of the test stopped before performing the check. It's a bit tricky because this https://jestjs.io/docs/en/expect#tothrowerror doesn't work like that with async/await. Anyway, I found a better and shorter solution:
`await expect(citeArticle(format, 12345)).rejects.toThrow(new Error('Request failed with status code 500'));`
which I'll push now.